plugins/index.js

```diff
@@ -22,7 +22,15 @@ module.exports = (on, config) => {
     // `on` is used to hook into various events Cypress emits
     // `config` is the resolved Cypress config
 
-    const { baseUrl, testName } = config.env;
+    const { integrationFolder, testFiles, baseUrl, testName } = config.env;
+
+    if (integrationFolder) {
```

Comment:
Was that a requirement? If I'm right, this will support `CYPRESS_integrationFolder=dir` command line environment variables.

Reply:
Yes, the command `npm run test` doesn't resolve the paths configured in `cypress.env.json` unless these params are set in the plugins file.